runtime/binding-kafka/src/main/zilla/internal.idl

```diff
@@ -36,6 +36,7 @@ scope internal
             kafka::KafkaHeader[] trailers;
             uint32 paddingLen;
             octets[paddingLen] padding;
+            int64 filters;
```

Comment:
This change needs to be reverted as `KafkaCacheEntry` is shared across multiple readers, whereas `filters` mask applies to each individual cache client stream.